docs/source/concepts/time.rst

```diff
@@ -0,0 +1,90 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _time:
+
+Time
+####
+
+The DAML language contains a function :ref:`getTime <daml-ref-gettime>` which returns the “current time”.
+However, the notion of time comes with a lot of problems in a distributed setting.
```

Comment:
Maybe worth at least hinting at some of the problems for users that aren’t too familiar with this?

Reply:
Probably yes, but I don't feel comfortable with stating any issues. Do you have suggestions? Just a generic "clock skew, synchronization, and if you solve these you always have space time from theory of relativity?"

Reply:
Not sure I have great suggestions so maybe just leave it out for now. We can always add it later.